lib/chef/knife/vsphere_pool_list.rb

```diff
@@ -15,7 +15,7 @@ class Chef::Knife::VspherePoolList < Chef::Knife::BaseVsphereCommand
   def traverse_folders(folder)
     return if folder.is_a? RbVmomi::VIM::VirtualApp
 
-    puts "#{ui.color("Pool", :cyan)}: "+(folder.path[3..-1].map { |x| x[1] }.* '/') if
+    puts "#{ui.color("Pool", :cyan)}: "+(folder.path[3..-1].map { |x| x[1] unless x[1] == "Resources" }.* '/').squeeze("/") if
```

Comment:
No, sorry. I'm just happy that it works. ;-)

```
if folder.is_a? RbVmomi::VIM::ResourcePool
  pools = folder.path[3..-1].reject { |p| p.last == "Resources" }
  puts "#{ui.color("Pool", :cyan)}: " + pools.map(&:last).join('/')
end
```

Changed ```first``` to ```last``` but the rest is great.